packages/vue/src/components/authenticator.vue

```diff
@@ -107,6 +107,12 @@ unsubscribeMachine = service.subscribe((newState) => {
     });
     hasInitialized.value = true;
   }
+  const facadeValues = getServiceFacade({
+    send,
+    state: newState,
+  });
+
+  updateValues(facadeValues);
```

Comment:
Now we will rely on on the xState subscription to update state, instead of a watch. This will reduce the number of times we update the `useAuthenticator` reactive state, and increase performance.